static/css/mozsearch.css

```diff
@@ -274,6 +274,17 @@ body {
 }
 /* Search results */
 
+/* Make context lines more subtle. */
+.results tr.before-context-line,
+.results tr.after-context-line {
+  color: gray;
```

Comment:
Maybe just `opacity: .5` would work with dark mode as well?

Reply:
Yeah, I think that even works better since it also makes the context line numbers more subtle too.